### PR TITLE
Adjusted test assertions with Kotlin infix functions.

### PR DESCRIPTION
**Pull request by antoniq**

### CODE REVIEW

File: library-integration-slack/src/test/kotlin/utils/TestAssertions.kt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package utils
+
+import org.assertj.core.api.Assertions
+import org.assertj.core.api.ListAssert
+import org.assertj.core.api.ObjectAssert
+
+/** Generic assertions for AssertJ */
+
+infix fun <T> T.assertsEqualTo(expected: T): ObjectAssert<T> = Assertions
```
Comment (slu-it) [inline library-integration-slack/src/test/kotlin/utils/TestAssertions.kt:9]: Given that the infix function changes how the assertion statement is read, I would suggest a more sentence like function name like `shouldBeEqualTo`. There is also a good Kotlin testing library which provides such methods `io.kotlintest:kotlintest-assertions:3.4.2` (https://github.com/kotest/kotest)

File: library-integration-slack/src/test/kotlin/utils/TestAssertions.kt [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    .assertThat(this)
+    .isEqualTo(expected)
+
+infix fun <T> List<T>.assertsContainsOnly(expected: T): ListAssert<T> = Assertions
```
Comment (slu-it) [inline library-integration-slack/src/test/kotlin/utils/TestAssertions.kt:13]: see above comment

File: library-integration-slack/src/test/kotlin/utils/TestAssertions.kt (hunk 1 above)
Comment (antoniq) [reply library-integration-slack/src/test/kotlin/utils/TestAssertions.kt:9]: Yes, my idea was to show how exisiting assertions with AssertJ can be wrapped into infix functions.  The Kotest library has definately more powerful methods. Should I leave a couple of tests only with the custom infix and integrate the rest with kotest lib?

Comment (antoniq) [reply library-integration-slack/src/test/kotlin/utils/TestAssertions.kt]: Should syntax style is updated.

**Pull request merged**